src/modules/transaction/components/TxBroadcaster/illustrationsMap.js

```diff
@@ -1,27 +1,32 @@
 import { txStatusTypes } from '@transaction/configuration/txStatus';
-import { getDeviceType } from '@wallet/utils/hwManager';
 
 const illustrations = {
   default: {
     [txStatusTypes.signatureSuccess]: 'transactionPending',
     [txStatusTypes.broadcastSuccess]: 'transactionSuccess',
     [txStatusTypes.broadcastError]: 'transactionError',
     [txStatusTypes.signatureError]: 'transactionError',
-    [txStatusTypes.hwRejected]: 'HwRejection',
+    [txStatusTypes.hwRejected]: 'hwRejection',
+    [txStatusTypes.hwDisconnected]: 'hwRejection',
+    [txStatusTypes.hwLiskAppClosed]: 'hwRejection',
   },
   stake: {
     [txStatusTypes.signatureSuccess]: 'stakingSuccess',
     [txStatusTypes.broadcastSuccess]: 'stakingSuccess',
     [txStatusTypes.broadcastError]: 'transactionError',
     [txStatusTypes.signatureError]: 'transactionError',
-    [txStatusTypes.hwRejected]: 'HwRejection',
+    [txStatusTypes.hwRejected]: 'hwRejection',
+    [txStatusTypes.hwDisconnected]: 'hwRejection',
+    [txStatusTypes.hwLiskAppClosed]: 'hwRejection',
```

Comment:
for all types we are classifying them as `hwRejection`?

Reply:
Each hw tx error has its own `txStatusType`, however, all use the same error icon

Reply:
hmmm ok.. @Ade-kola could have given different icons if needed